.travis.yml

```diff
@@ -6,25 +6,32 @@ sudo: 		required
 dist: 		trusty
 git:
   submodules:   false
-before_install:
-  - sudo apt-get update -qq
-install:
-  - sudo apt-get install -y ocaml ocaml-nox ocaml-native-compilers ocaml-findlib
-  - sudo apt-get install -y camlp5 camlp4-extra time libgtk2.0 libgtksourceview2.0
-  - sudo apt-get install -y liblablgtk-extras-ocaml-dev
-  # get etags from emacs:
-  - sudo apt-get install -y emacs
+cache:
+  apt: true
```

Comment:
What does this line do?

Reply:
[nothing](https://github.com/travis-ci/travis-ci/issues/5876), apparently.

Reply:
How about removing lines 9-10, in that case?

Reply:
Done!